class-list-table.php

```diff
@@ -0,0 +1,131 @@
+<?php
+/**
+ * List table for displaying the list of sites.
+ *
+ * @package Pantheon_Decoupled
+ */
+
+use WP_List_Table;
+
+/**
+ * List table for displaying the list of sites.
+ */
+class List_Table extends WP_List_table {
```

Comment:
Using the preview list table would make the class definition something like:

```
class FES_Preview_Table extends List_Table {
}
```

We could have just referenced this class in pantheon_decoupled_preview_list_html directly - at this point I don't think we actually need to customize any functions. I think stubbing out this custom class now makes sense though because we will customize it in the future.